linera-storage-service/src/server.rs

```diff
@@ -661,10 +667,11 @@ async fn main() {
                 inner_config,
                 storage_cache_config,
             };
-            let store = RocksDbStore::maybe_create_and_connect(&config, &namespace)
+            let database = RocksDbDatabase::maybe_create_and_connect(&config, &namespace)
                 .await
                 .expect("store");
-            let store = LocalStore::RocksDb(store);
+            let store_impl = database.open_shared(&[]).expect("Failed to open store");
```

Comment:
Usually, we do not use stuff like `store_impl` where the `_impl` does not add anything.

Reply:
thx. I missed that one